lib/puppet_forge/v3/release.rb

```diff
@@ -27,7 +27,7 @@ def download(path)
         resp = self.class.conn.get(download_url)
         path.open('wb') { |fh| fh.write(resp.body) }
       rescue Faraday::ResourceNotFound => e
-        raise PuppetForge::ReleaseNotFound, "The module release #{slug} does not exist on #{self.class.conn.url_prefix}.", e.backtrace
+        raise PuppetForge::ReleaseNotFound, _("The module release %{release_slug} does not exist on %{url}.%{backtrace}") % {release_slug: slug, url: self.class.conn.url_prefix, backtrace: e.backtrace}
```

Comment:
backtrace is an argument to raising the exception, not an interpolated value here. Unless that somehow falls apart with externalization?